### DISCUSSION TITLE
params object

### DISCUSSION THREAD

**Discussion by zms9110750** [dotnet/csharplang#8851] [Language Ideas]
When using default parameters, we may overload many methods due to quantity and order.
Finally, we make an Options type as a method parameter passed in.

I hope that if params is used to modify an object,
We can consider his members as default parameters for this method
```
void Serialize<TValue>(TValue value,params JsonSerializerOptions options)
{
}
```
```
Serialize(value, PropertyNameCaseInsensitive = true);
```

**Comment by CyrusNajmabadi** [comment]
This is aready possible with `new() { PropertyNameCaseInsensitive = true }` where more named properties can be added after this.
